pkgs/development/python-modules/macholib/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchPypi, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "macholib";
+  version = "1.14";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-DENryEfnsdm9oFYDUb9218r5MPtYWoKNE2CIOe9CxDI=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ altgraph ];
+
+  doCheck = false;
```

Comment:
pythonImportsCheck is required when disabling tests and also add a comment why you disabled tests.